components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1269,16 +1300,22 @@ - (void)setMinimumHeight:(CGFloat)minimumHeight {
 }
 
 - (void)setMaximumHeight:(CGFloat)maximumHeight {
-  _hasExplicitlySetMaxHeight = YES;
+  _hasExplicitlySetMinOrMaxHeight = YES;
   if (_maximumHeight == maximumHeight) {
     return;
   }
 
   CGPoint originalOffset = _trackingScrollView.contentOffset;
-
   [self fhv_removeInsetsFromScrollView:_trackingScrollView];
 
   _maximumHeight = maximumHeight;
+  NSLog(@"%f", _maximumHeight);
```

Comment:
Delete.

Reply:
it wasn't ready for review just yet :P (notice the isPlanningChanges label)

Reply:
Ah! Sorry 😅